### PR TITLE
Fix variable names - add _ to names starting with a numeric

### PR DESCRIPTION
**Pull request by neinoi**
Fix proposition for issue #319

### CODE REVIEW

File: multiapi-engine/src/main/resources/templates/asyncapi/templateSchema.ftlh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <#assign words = ["abstract", "assert", "boolean", "break", "byte", "case", "catch", "char", "class", "const", "continue", "default", "double", "do", "else", "enum", "extends", "false", "final", "finally", "float", "for", "goto", "if", "implements", "import", "instanceof", "int", "interface", "long", "native", "new", "null", "package", "private", "protected", "public", "return", "short", "static", "strictfp", "super", "switch", "synchronized", "this", "throw", "throws", "transient", "true", "try", "void", "volatile", "while"]>
 <#function calculateSafeName propertyName ender>
-    <#if words?seq_contains(propertyName?uncap_first)>
+    <#if words?seq_contains(propertyName?uncap_first) || propertyName?matches("[0-9].*")>
```
Comment (jemacineiras) [inline multiapi-engine/src/main/resources/templates/asyncapi/templateSchema.ftlh:3]: I love it. 
Same fix should be done for Openapi files.
Also we use semver to change product versión number. So if you can bump lower version number and also update the Readme.md I think we can approve the merge.

**Review by jemacineiras — CHANGES_REQUESTED** [review]

File: multiapi-engine/src/main/resources/templates/asyncapi/templateSchema.ftlh (hunk 1 above)
Comment (neinoi) [reply multiapi-engine/src/main/resources/templates/asyncapi/templateSchema.ftlh:3]: Done

Comment (neinoi) [reply multiapi-engine/src/main/resources/templates/asyncapi/templateSchema.ftlh:3]: I think I’ve fixed the issues for enums, there was more templates parts to update.

Comment (neinoi) [reply multiapi-engine/src/main/resources/templates/asyncapi/templateSchema.ftlh:3]: It should fix openapi in the same time, I runned the tests to validate I don’t break existing code.

**Review by jemacineiras — APPROVED** [review]
We usually add a test plan for every case, or modify an existing one to handle it. But I will made an exception here.

**Comment by neinoi** [comment]
> We usually add a test plan for every case, or modify an existing one to handle it. But I will made an exception here.

I’ll try to add it later, I don’t have the time to do it now.

**Comment by neinoi** [comment]
Hi @jemacineiras , how much time does it take to publish the plugin in public repositories ? I still do not see it …

**Pull request merged**